src/App/App.module.css

```diff
@@ -4,16 +4,53 @@ body {
   margin: 0;
 }
 
-.wrap {
-  margin: 0 auto;
+a {
+  text-decoration: none;
+  }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5932591)